### ISSUE TITLE
Notion enhancer won't install "you do not have permission" or "files don't exist"

### ISSUE THREAD

**Issue by tryuut89** [notion-enhancer/notion-enhancer#465]
FILL OUT THIS TEMPLATE, REMOVING/REPLACING ALL NON-BOLD TEXT WITH YOUR OWN INFORMATION.
BE AS CLEAR AND CONCISE AS POSSIBLE.

**describe the bug**
cannot get it to install for the life of me, may be a simple issue, but it just won't load in, it either says "you do not have permission" or "files don't exist". I've seen lots of similar issues being resolved on here but those fixes don't work for me either? like the updated code for the new notion, or taking out lines and replacing them with other code, it just doesn't work (its probably me i know NOTHING about coding)

**expected behavior**
it should install and load into my computer

**screenshots**
if applicable, add screenshots of your problem.

**platform**
MAC OS big Sur 11.2.3, newest notion desktop app

**Comment by dragonwocky** [comment]
please provide screenshots of the errors, the context will be important for solving this issue!

**Comment by tryuut89** [comment]
> i

Ahh okay sorry i'll retry and add them!

**Comment by tryuut89** [comment]
> please provide screenshots of the errors, the context will be important for solving this issue!

I added a screenshot however my issue changed!

**Comment by dragonwocky** [comment]
it looks like there's something wrong with your node.js installation, try uninstall and reinstalling that.

**Comment by tryuut89** [comment]
> it looks like there's something wrong with your node.js installation, try uninstall and reinstalling that.

thank you, how do I find it? it doesn't come up when I search in my spotlight

**Comment by dragonwocky** [comment]
well that's probably it issue, it must not be installed at all. try installing from https://nodejs.org/en/download/

**Comment by tryuut89** [comment]
> well that's probably the issue, it must not be installed at all. try installing from https://nodejs.org/en/download/

thank you, I ran npm i -g notion-enhancer and this happened, is it installed now?
<img width="362" alt="Screenshot 2021-04-17 at 17 56 50" src="https://user-images.githubusercontent.com/82719070/115120624-5e468700-9fa6-11eb-9f28-3c9d1e27ed84.png">

**Comment by tryuut89** [comment]
> well that's probably the issue, it must not be installed at all. try installing from https://nodejs.org/en/download/

this is what it says now 
<img width="253" alt="Screenshot 2021-04-17 at 17 59 42" src="https://user-images.githubusercontent.com/82719070/115120702-bc736a00-9fa6-11eb-8ef4-302728017684.png">

**Comment by tryuut89** [comment]
i ran code from a youtube video to fix the above issue but now it doesn't open at all when clicked (although it says its open in the dock) i tried using the code I linked as I saw someone say that helped fixed my issue but when I run notion-enhancer apply -y, this is all that comes up, so there Is no code to replace
<img width="299" alt="Screenshot 2021-04-17 at 18 16 09" src="https://user-images.githubusercontent.com/82719070/115121136-01000500-9fa9-11eb-9873-78fccb5b0cd2.png">

**Comment by tryuut89** [comment]
i ran code from a youtube video to fix the above issue but now it doesn't open at all when clicked (although it says its open in the dock) i tried using the code I linked as I saw someone say that helped fixed my issue but when I run notion-enhancer apply -y, this is all that comes up, so there Is no code to replace
![Uploading Screenshot 2021-04-17 at 18.29.33.png…]()

**Comment by tryuut89** [comment]
> well that's probably the issue, it must not be installed at all. try installing from https://nodejs.org/en/download/
i ran code from a youtube video to fix the above issue but now it doesn't open at all when clicked (although it says its open in the dock) i tried using the code I linked as I saw someone say that helped fixed my issue but when I run notion-enhancer apply -y, this is all that comes up, so there Is no code to replace
<img width="315" alt="Screenshot 2021-04-17 at 18 30 36" src="https://user-images.githubusercontent.com/82719070/115121503-02323180-9fab-11eb-88b4-ca60dd8c7b8b.png">

**Comment by dragonwocky** [comment]
try running `codesign --force --deep --sign - /Applications/Notion.app` in terminal

**Comment by tryuut89** [comment]
> try running `codesign --force --deep --sign - /Applications/Notion.app` in terminal

still won't open:/

**Comment by dragonwocky** [comment]
screenshot of what happens when you run that please?

**Comment by tryuut89** [comment]
<img width="388" alt="Screenshot 2021-04-18 at 17 43 33" src="https://user-images.githubusercontent.com/82719070/115153404-995fbd80-a06d-11eb-8193-6c565c7f5bf1.png">
sorry sure! it seems to run and work, but not solve the issue

**Comment by Jasminecbrown91** [comment]
![Screen Shot 2021-04-21 at 3 45 06 PM](https://user-images.githubusercontent.com/78446050/115630277-99073280-a2b8-11eb-898c-1aca6cdf81dc.png)


This is what I get after running "codesign --force --deep --sign - /Applications/Notion.app"....Notion seems to have permissions issues and not open after I restart my mac. I'm on Big Sur 11.2.3 with latest version of Notion & this version of Node.js (Latest Current Version: 16.0.0 (includes npm 7.10.0

**Comment by dragonwocky** [comment]
@Jasminecbrown91 try the fix here: https://apple.stackexchange.com/questions/254380/why-am-i-getting-an-invalid-active-developer-path-when-attempting-to-use-git-a

**Comment by emchi-31** [comment]
i am wondering if "make sure that /usr/local/bin is in your $PATH" while installing node.js has any impact for our problem?

**Comment by dragonwocky** [comment]
@emchi-31 only if you get the `not found` in terminal, its unrelated to the permissions issue

**Comment by nandudb** [comment]
i do see same issue after updating big sur 11.3
![image](https://user-images.githubusercontent.com/8333356/116632375-0d7b4a80-a925-11eb-8cc9-90fc550c8038.png)
![image](https://user-images.githubusercontent.com/8333356/116632425-34398100-a925-11eb-9599-be962d18138d.png)

**Comment by emchi-31** [comment]
> <img alt="Screenshot 2021-04-18 at 17 43 33" width="388" src="https://user-images.githubusercontent.com/82719070/115153404-995fbd80-a06d-11eb-8193-6c565c7f5bf1.png">
> 
> sorry sure! it seems to run and work, but not solve the issue

have u been able to use the enhancer?
@tryuut89

**Comment by tryuut89** [comment]
Nope, I ended up deleting it and using the standard version since nothing worked :/ I’ve tried installing again but one of the above issues always comes up

**Comment by tatumdale** [comment]
Did you do the reinstall to refresh Notion and then login before re-adding the Notion_enhancer? @tryuut89

**Comment by tryuut89** [comment]
Yep I tried that (although it auto logged me in) and it still didn’t work :/

Regards,

Carl

**Comment by HassanMojab** [comment]
I have the same issue.
![image](https://user-images.githubusercontent.com/29512354/118411532-4ac43400-b6aa-11eb-9bcf-1ed027bd507e.png)

**Comment by HassanMojab** [comment]
> Did you do the reinstall to refresh Notion and then login before re-adding the Notion_enhancer? @tryuut89

It worked!
I uninstalled/reinstalled the app and this time before applying the `notion-enhancer`, I opened the app and it logged in to my account automatically, then closed it and ran the `notion-enhancer apply`.

**Comment by tryuut89** [comment]
I’ll try that out thank you!

Regards,

Carl

**Comment by dragonwocky** [comment]
if anyone is still experiencing this error, `xattr -cr /path/to/application.app` seems to fix it
